charts/timed/values.yaml

```diff
@@ -67,5 +71,8 @@ ingress:
     #   hosts:
     #     - chart-example.local
 postgresql:
-  install: true
+  enabled: true
   postgresqlDatabase: timed
+  postgresqlUsername: postgres
+  image:
+    tag: 12.2.0
```

Comment:
Is this our way of locking the used postgres version? Is that not already achieved by the way we specify the chart version dependency in `Chart.yaml`?

Reply:
the latest chart installs postgres 11.7.0: https://github.com/bitnami/charts/blob/master/bitnami/postgresql/Chart.yaml
this is the way proposed by bitnami: https://github.com/bitnami/charts/tree/master/bitnami/postgresql/#change-postgresql-version

Reply:
@hairmare What do you think? That way we specify the version in `Chart.yaml` we're going to stay within the minor release of the chart and I don't think Bitnami is going to update the postgres version without a bump in the chart version. On the other hand explicitly specifying the postgres image version gives us more flexibility.

Reply:
Let's keep it like this for now until we figure out if everything works fine.